constellations/orion/src/domains/transaction.go

```diff
@@ -0,0 +1,38 @@
+package domains
+
+import (
+	"errors"
+	"time"
+)
+
+var TABLE_TRANSACTIONS = "transactions"
+
```

Comment:
Here we use capital-case SNAKE_CASE because it's usually the way to signify that this is a static non-changing value that can be used in many places around the codebase